tests/unit/contentRecog/test_contentRecog.py

```diff
@@ -0,0 +1,155 @@
+#tests/unit/contentRecog/test_contentRecog.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2017 NV Access Limited
+
+"""Unit tests for the contentRecog module.
+"""
+
+import unittest
+import contentRecog
+import textInfos
+
+class TestResultCoordConverter(unittest.TestCase):
+
+	def test_noOffsetNoResize(self):
+		conv = contentRecog.ResultCoordConverter(0, 0, 1)
+		actual = conv.convertX(100), conv.convertY(200)
+		self.assertEqual(actual, (100, 200))
+
+	def test_withOffsetNoResize(self):
+		conv = contentRecog.ResultCoordConverter(10, 20, 1)
+		actual = conv.convertX(100), conv.convertY(200)
+		self.assertEqual(actual, (110, 220))
+
+	def test_noOffsetWithResize(self):
+		conv = contentRecog.ResultCoordConverter(0, 0, 2)
+		actual = conv.convertX(200), conv.convertY(400)
+		self.assertEqual(actual, (100, 200))
+
+	def test_withOffsetWithResize(self):
+		conv = contentRecog.ResultCoordConverter(10, 20, 2)
+		actual = conv.convertX(200), conv.convertY(400)
+		self.assertEqual(actual, (110, 220))
+
+class FakeNVDAObject(object):
+	pass
+
+class TestLinesWordsResult(unittest.TestCase):
+	"""Tests contentRecog.LinesWordsResult and contentRecog.LwrTextInfo.

```

Comment:
I think its worth expanding on this a little bit. This is essentially testing that we are able to parse and interpret the json we expect to be returned from an OCR library, right?